src/Tooltip/Tooltip.tsx

```diff
@@ -42,7 +42,7 @@ export enum TooltipPlacement {
 }
 
 export type TooltipProps = MuiTooltipProps & {
-  title?: string,
+  title?: React.ReactNode, // DXQ-37812-Updated to accept both string and HTML node
```

Comment:
please avoid referencing non-public ticket numbers in the opensource project.
```suggestion
  title?: React.ReactNode,
```

Reply:
Got it